include/crow/middlewares/cookie_parser.h

```diff
@@ -43,14 +166,17 @@ namespace crow
                 return {};
             }
 
-            void set_cookie(const std::string& key, const std::string& value)
+            template<typename U>
+            Cookie& set_cookie(const std::string& key, U&& value)
             {
-                cookies_to_add.emplace(key, value);
+                cookies_to_add.emplace_back(key, std::forward<U>(value));
+                return cookies_to_add.back();
             }
         };
 
         void before_handle(request& req, response& res, context& ctx)
         {
+            // TODO: remove copies, use string_view with c++17
```

Comment:
it would be ideal if you change this to `TODO(dranikpg):` so later on we have a bit more context on it.